src/bigint.rs

```diff
@@ -809,100 +809,68 @@ impl_to_biguint!(u16,  FromPrimitive::from_u16);
 impl_to_biguint!(u32,  FromPrimitive::from_u32);
 impl_to_biguint!(u64,  FromPrimitive::from_u64);
 
-// Cribbed from core/fmt/num.rs
-#[derive(Copy, Clone)]
-pub struct RadixFmt {
```

Comment:
Strange that this was `pub`, but I don't see any way for users to create or receive one.  I guess someone *could* have written functions taking `RadixFmt` as parameters, but they would be useless (can only `Clone`/`Copy`/`Display`) and have no callers.  This is still an API break if we're being ultra-cautious, but I think it's ok.